### PR TITLE
Bookstore: Api

### PR DESCRIPTION
**Pull request by ivaan1992**
For this project I: 

- Read API documentation.
- Created a unique ID
- Make add and remove books function into the API.

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @ivaan1992, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Great job with Gitflow

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and under the review._

### I will leave this comment here because you did not make any change in your `readme file` for this particular `PR` thus it is not accessible.

- [x] Please kindly update your `readme file` with information pertaining to this particular milestone. Checkout this [template](https://github.com/microverseinc/readme-template) and adapt it to your project. This will make your `readme file` more informative and professional. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js [hunk 1]
```diff
@@ -13,13 +13,18 @@ const Form = () => {
...
+  };
   const handleAddBook = () => {
     const newBook = {
       name: title,
       author,
+      category,
       id: Math.ceil(Math.random() * 10000),
     };
-    dispatch(addBook(newBook));
+    dispatch(addBooksAsync(newBook));
```
Comment (medaminedev66) [inline src/components/Form.js:27]: - [x] I tried to add a book. When I reloaded the page, I was not able to find the book that I added. Please make sure that the book is sent to the server using the API after clicking on the `Add book` button, and all the books should be fetched from the API while loading the page.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @ivaan1992, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️There are no linter errors
✔️You have a descriptive PR

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ivaan1992 , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**